### PR TITLE
Allow FosJsRouting dev for Symfony 3

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

**Comment by WouterJ** [comment]
Build is fixed again

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
         "symfony-cmf/testing": "^1.3@dev",
         "symfony/dependency-injection": "^2.0.5|3.*",
         "symfony/config": "^2.2|3.*",
-        "symfony/event-dispatcher": "^2.1|3.*"
+        "symfony/event-dispatcher": "^2.1|3.*",
+        "friendsofsymfony/jsrouting-bundle": "@dev"
```
Comment (dbu) [inline composer.json]: do we require this or is it some other bundle that requires it? then the `@dev` should go in there rather than here, i think.

Comment (WouterJ) [inline composer.json]: Testing requires it. `@dev` isn't needed, only to support Symfony 3 as a Symfony 3 supporting version has not been released.

However, as minimum stability is now set to stable for all CMF repositories, we need to explicitely allow FOSJsRoutingBundle to be installed in dev when needed.

Comment (dbu) [inline composer.json]: okay, i see. i hope to release this library really soon - should we do so with this line or should we try to tag a stable version of jsrouting?

Comment (dbu) [inline composer.json]: ah its only a dev dependency so no big deal!

Comment (WouterJ) [inline composer.json]: Yeah and it's a dep that's not even used by this library. This is one of the problems of Testing 1.0, eventhough this is a component tested without framework integration, many bundles are still automatically loaded by Testing. For Testing 2.0, I would like to remove all config/required bundles from the component, so each library can make choices themselves.

**Pull request merged**